shared/util/debug.tsx

```diff
@@ -1,4 +1,4 @@
-export const ENABLE_F5_REMOUNTS = __DEV__ && (true as boolean)
+export const ENABLE_F5_REMOUNTS = __DEV__ && (false as boolean)
```

Comment:
shouldn't be on in master likely and the changes in stylesheetproxy cause rerender issues so turn this off. this is fixed in a better way in the next mr